labelspark/__init__.py

```diff
@@ -39,6 +39,39 @@ def get_annotations(client, project_id, spark, sc):
     return bronze_table
 
 
+import requests  #we need to handle a large frame-by-frame dataset for videos, so we use requests
+
+
+def get_videoframe_annotations(bronze_video_labels, API_KEY, spark, sc):
```

Comment:
I would not name the argument `API_KEY` I would name it at least `api_key`


`SCREAMING_CASE` is generally reserved for constants as a opposed to parameters

Reply:
Adjusted to api_key